cmd/govim/main.go

```diff
@@ -428,3 +435,29 @@ func (g *govimplugin) Shutdown() error {
 	}
 	return nil
 }
+
+func (g *govimplugin) defineHighlights() {
+	warnColor := 166 // Orange
+	if vimColors, err := strconv.Atoi(g.ParseString(g.ChannelExpr(`&t_Co`))); err != nil || vimColors < 256 {
+		warnColor = 3 // Yellow, fallback when the terminal doesn't support at least 256 colors
+	}
+	g.ChannelExf("highlight default %s term=underline cterm=underline gui=undercurl ctermfg=1 guisp=Red", types.HighlightErr)
```

Comment:
Batch these calls? Not critical if we don't, but will help to reduce startup time by a few ms.